source/commands.txt

```diff
@@ -0,0 +1,163 @@
+.. _vsce-commands:
+
+============================
+MongoDB for VS Code Commands
+============================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+This document lists the VS Code commands available for
+|vsce|. To open the Command Palette and execute these commands:
+
+.. include:: /includes/list-tables/command-palette-options.rst
+
+.. include:: /includes/admonitions/tip-mongodb-command-palette.rst
+
+Connection Commands
+-------------------
+
+Run these commands from the Command Palette to create new connections or
+manage existing connections to your MongoDB deployments.
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 40
+
+   * - Command
+     - Description
+
+   * - :ref:`Connect <vsce-cmd-connect>`
+
+       .. _vsce-cmd-connect:
+
+     - Connect to your MongoDB deployment by filling in your connection
+       information in the connection dialog. To learn more about
+       connecting to a deployment, see :ref:`vsce-connect`.
+
+   * - :ref:`Connect with Connection String <vsce-cmd-connect-string>`
+
+       .. _vsce-cmd-connect-string:
+
+     - Connect to your MongoDB deployment by pasting your 
+       :manual:`connection string URI </reference/connection-string/>`
+       directly into the command palette.
+
+       |service| provides a connection string for your clusters which
+       you can pass into this command.
+       
+       To learn more about connecting to a deployment, see
+       :ref:`vsce-connect`.
+
+   * - :ref:`Disconnect <vsce-cmd-disconnect>`
+
+       .. _vsce-cmd-disconnect:
+
+     - If you are currently connected to a MongoDB deployment,
+       disconnect from that deployment.
+
+   * - :ref:`Remove Connection <vsce-cmd-remove-connection>`
+
+       .. _vsce-cmd-remove-connection:
+
+     - Select and remove a connection from your list of connections
+       in |vsce|.
+
+.. note::
+
+   You can only have one active connection to a MongoDB
+   deployment open at a time.
+
+Shell Commands
+--------------
+
+Run this command from the Command Palette to open a MongoDB Shell
+connected to your deployment.
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 40
+
+   * - Command
+     - Description
+
+   * - :ref:`Launch MongoDB Shell <vsce-cmd-launch-shell>`
+
+       .. _vsce-cmd-launch-shell:
+
+     - .. important::
+
+          This command is not supported on Windows.
+     
+       Open the VS Code :guilabel:`Terminal` and automatically connect a
+       MongoDB shell to the deployment where |vsce| is connected. You
+       can set which shell this command uses in your ``mdb.shell``
+       :ref:`setting <vsce-settings>`.
```

Comment:
Maybe merge the second sentence with 102, like:

"Specify the MongoDB shell to use in the ``mdb.shell`` setting. The MongoDB |vsce| extension supports the following shells:"

And then maybe include an example string for how to specify that shell (is it just `mongo` or `mongosh` ? Like, the value should be the exact string you would pass into the terminal?

Reply:
Agreed on the merge. The setting is a dropdown, so I think no clarification necessary.